src/main/java/net/dv8tion/jda/core/requests/restaction/RoleOrderAction.java

```diff
@@ -25,10 +25,27 @@
 
 import java.util.List;
 
+/**
+ * Implementation of {@link net.dv8tion.jda.core.requests.restaction.OrderAction OrderAction}
+ * designed to modify the order of {@link net.dv8tion.jda.core.entities.Role Roles} of the
+ * specified {@link net.dv8tion.jda.core.entities.Guild Guild}.
+ * <br>To apply the changes you must finish the {@link net.dv8tion.jda.core.requests.RestAction RestAction}
+ *
+ * <p><b>This uses descending order!</b>
```

Comment:
Example of descending order would probably be good to add. E.g: "Positions are descending, meaning that the highest Role in the Guild is at `n-1` in the list, which is based on Role#getPosition` or something.